### ISSUE TITLE
area type: "snow"

### ISSUE THREAD

**Issue by taylor13** [cf-convention/discuss#147]
In the [CF Area Type Table](https://cfconventions.org/Data/area-type-table/10/build/area-type-table.html), "snow" is listed without any further explanation of its meaning.  Also listed in the table is "rain", with the explanation that ``An area_type of "rain" indicates that falling rain is present at some level in the atmospheric column above an area on the surface of the Earth.``  My question is: are the "snow" and "rain" area types supposed to be interpreted similarly?

If so, how would one record the surface temperature of the snow?  I note that the standard names table describes surface_temperature as follows:
```
The surface called "surface" means the lower boundary of the atmosphere. The surface temperature 
is the temperature at the interface, not the bulk temperature of the medium above or below. Unless
 indicated in the cell_methods attribute, a quantity is assumed to apply to the whole area of each 
horizontal grid box. Previously, the qualifier where_type was used to specify that the quantity applies 
only to the part of the grid box of the named type. Names containing the where_type qualifier are 
deprecated and newly created data should use the cell_methods attribute to indicate the horizontal 
area to which the quantity applies.
```
If we interpret the "snow" area type as the surface area covered by snow, then we could use surface_temperature with "where snow" in the ``cell_methods`` attribute, but if we interpret "snow" in the sense that "rain" is interpreted, then this would be incorrect.  

Perhaps, we should define another area type as "surface_snow", with an obvious interpretation, and also add an explanation of "snow" that is similar to the description of the "rain" area type.

**Comment by DanHollis** [comment]
We don't have a need for the surface temperature of snow but we do create datasets of the number of days with snow lying on the ground. For this we use the standard name `surface_snow_binary_mask` with `surface_snow_area_fraction: 0.5`.

However I agree that the meaning of area type `snow` should be clarified. I see that there are a couple of related area types - `ice_and_snow_on_land` and `snow_free_land`. In these two examples the word 'snow' is being used to mean surface snow. Following this pattern, another option would be to add `snow_on_land`. Having said that, there are quite a few existing standard names that include 'surface_snow' in the name, but also some that refer to just 'snow'. I haven't attempted to work out whether the latter refer to falling snow or lying snow. My vote would be to define the existing `snow` area type as falling snow (to match `rain`) and add either `surface_snow` or `snow_on_land`.

**Comment by taylor13** [comment]
just to note: "snow_on_land" would not be the same as "surface_snow" because snow can lie on sea ice and land ice too, which are (in CF) not considered to be "land". 

Unless anyone knows that in the past "snow" has been interpreted as "falling_snow", as opposed to "surface_snow", we might prefer "snow" to mean "surface_snow", since I know for a fact that it has commonly been used to mean that in the past.

**Comment by taylor13** [comment]
Does anyone object to defining the standard_name for area_type "snow" as "snow lying on the surface" (not "snow residing or falling in the atmosphere"?

**Comment by JonathanGregory** [comment]
Perhaps it's snow lying on the surface or the canopy?

**Comment by taylor13** [comment]
Yes, probably should be.  So "snow lying on the surface or collected by the canopy"

**Comment by davidhassell** [comment]
I have no prior experience of using this area type, but having read this thread, I would have no objection to redefining area type "snow" to mean the suggested "snow lying on the surface or collected by the canopy" (although I'm not sure what is meant by _the standard_name for area_type "snow"_   - was that intended? apologies if I'm missing something obvious!).

**Comment by taylor13** [comment]
Sorry, I think I misread something in one of the documents.  As I now understand it, the area_type identifiers are _not_ standard_names.  Please note, I don't think we're really redefining area type "snow" since it was never precisely stated before how it should be interpreted; we're just clarifying.

**Comment by taylor13** [comment]
Dear Alison @japamment (or whoever might be able to implement this)

There seems to be no objection and some support for clarifying the meaning of "snow" as an "area type" so in the drop down definition under "snow" in the [table](https://cfconventions.org/Data/area-type-table/current/build/area-type-table.html), could we replace "No help available" with "The portion of the surface covered by snow or with a canopy that hosts snow"?
